config/ignition_fortress_ubuntu_focal.yaml

```diff
@@ -69,11 +58,6 @@ filter_formula: "\
  Package (= libignition-physics5-tpelib) |\
  Package (= libignition-physics5-tpelib-dev) \
 ), $Version (% 5.1.0-2*) |\
-(Package (= ignition-plugin) |\
- Package (= libignition-plugin) |\
- Package (= libignition-plugin-dbg) |\
- Package (= libignition-plugin-dev) \
-), $Version (% 1.2.1-1*) |\
```

Comment:
I would prefer that these changes are revered for the reasons described in https://github.com/ros-infrastructure/reprepro-updater/pull/168#issuecomment-1218663623

Reply:
Reverted [adf0dce](https://github.com/ros-infrastructure/reprepro-updater/pull/168/commits/adf0dcee76545baaff671047521d91dad91e8176)